content-src/components/DiscoveryStreamComponents/Hero/_Hero.scss

```diff
@@ -268,7 +271,8 @@ $card-header-in-hero-line-height: 20;
             color: $white;
           }
 
-          @include limit-visibile-lines(3, 20, 14);
+          line-height: 20px;
+          font-size: 14px;
```

Comment:
These also seem to match up with `$card-header-in-hero-line-height` but not sure if they should use the existing variables ?